manual/overview.wiki

```diff
@@ -23,6 +23,11 @@ Js_of_ocaml is composed of multiple packages:
   * js_of_ocaml-toplevel, lib and tools to build an ocaml toplevel to
     javascript.
 
+Note: All code examples in this manual use Js_of_ocaml's ppx syntax.
+It is, however, possible to use Js_of_ocaml purely as a compiler
+while using a different library (e.g. gen_js_api, brr) to
```

Comment:
gen_js_api is not a library it's a ppx.

Reply:
Thank you, Daniel. Changed library to package.